examples/package.json

```diff
@@ -13,8 +13,8 @@
     "bem-components": "git+ssh://git@github.com/awinogradov/bem-components.git#v2",
     "bem-core": "git+ssh://git@github.com/bem/bem-core.git#v2",
     "bem-xjst": "git+ssh://git@github.com:awinogradov/bem-xjst.git#feature/bemreact",
-    "react": "^0.14.7",
-    "react-dom": "^0.14.7",
+    "react": "^15.1.0",
+    "react-dom": "^15.1.0",
```

Comment:
There was something weird with old react. React@14 created two nodes with the same reactid. Looks like a bug. But it  works with react@15